autoware.repos

```diff
@@ -75,6 +75,14 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/trt_batched_nms.git
     version: main
+  universe/external/cuda_blackboard:
+    type: git
+    url: https://github.com/autowarefoundation/cuda_blackboard.git
+    version: v0.1.1
+  universe/external/negotiated:
+    type: git
+    url: https://github.com/osrf/negotiated.git
+    version: master
```

Comment:
Could you point this to a version hash if possible?

Reply:
Addressed in https://github.com/autowarefoundation/autoware/pull/5710/commits/4b25221c42eb9a443e21da105df72d185901d1e0